components/match2/wikis/rainbow_six/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local wikiCopyPaste = {}
+
+local MODES = {
```

Comment:
doesn't match with _getOpponent function

Reply:
Converted to use /Base. Still using custom getOpponent func to have the option to disable the setting of |score=.